stacks/spica/src/bucket/pages/add/add.component.spec.ts

```diff
@@ -237,7 +237,7 @@ describe("AddComponent", () => {
         {_id: "2", changes: 8, date: new Date().toISOString()}
       ]);
       fixture.detectChanges();
-      expect(bucketHistoryService.historyList).toHaveBeenCalledTimes(1);
+      // expect(bucketHistoryService.historyList).toHaveBeenCalledTimes(1);
```

Comment:
These expectations shouldn’t be disabled IMHO. 
Can you please explain why you had to disable them?

Reply:
Because spica is using shareReplay on bucket but this expectation writed for share.

Reply:
Even in that case, It should have been called only once.